catalyst_voices/packages/internal/catalyst_voices_models/lib/src/feature_flags/feature_flag_source_type.dart

```diff
@@ -0,0 +1,12 @@
+/// Priority levels for feature flag sources (higher = higher priority)
+enum FeatureFlagSourceType {
```

Comment:
How about that name? `Type` doesn't bring much information anyway
```suggestion
enum FeatureFlagSource {
```